sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_utils.py

```diff
@@ -0,0 +1,22 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+from datetime import datetime
+from dateutil.parser import parse
+from msrest.serialization import TZ_UTC
```

Comment:
Please avoid msrest.
You should be able to use datetime.timezone.utc

Reply:
This have been rectified